dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -70,27 +132,28 @@ setup(){
     rosa whoami
     rosa verify quota
     rosa verify permissions
+    export ROSA_VERSION=$(rosa list versions -o json --channel-group=nightly | jq -r '.[] | select(.raw_id|startswith('\"${version}\"')) | .raw_id' | sort -rV | head -1)
```

Comment:
Does it make sense to capture a MAJOR_VERSION (4.9, 4.10) etc if we ever want to look holistically at major versions instead of the tiny details like minor version. If we see a major version regression then we can dig deeper into which nightly caused this. Thoughts?

Reply:
For sure. $version parameter is pointing to the major version, I can add it on the metadata function